middleware.ts

```
@@ -0,0 +1,18 @@
+import { NextRequest, NextResponse } from "next/server";
+import { auth } from "./lib/auth";
+import { headers } from "next/headers";
+
+export async function middleware(request: NextRequest, response: NextResponse) {
```

ESLint

[error] 'response' is defined but never used.

(@typescript-eslint/no-unused-vars)